Wrappers/Python/test/test_algorithms.py

```diff
@@ -395,6 +395,41 @@ def setup(data, dnoise):
             print ("RMSE", rmse)
         self.assertLess(rmse, 2e-4)
 
+    def test_PDHG_strongly_convex(self):
+
+        ig = ImageGeometry(3,3)
+        data = ig.allocate('random')
+
+        f = L2NormSquared(b=data)
+        g = L2NormSquared()
+        operator = IdentityOperator(ig)
+
+        sigma = 1.0
+        tau  = 1.0
+
+        pdhg = PDHG(f=f, g=g, operator=operator, max_iteration=10)
+        pdhg.run(verbose=0)
+        self.assertEqual(pdhg.sigma, sigma)
+        self.assertEqual(pdhg.tau, sigma)
+
+        pdhg = PDHG(f=f, g=g, operator=operator, max_iteration=10, gamma_g=0.5)
+        pdhg.run(verbose=0)
+        self.assertNotEqual(pdhg.sigma, sigma)
+        self.assertNotEqual(pdhg.tau, tau)       
+
+        pdhg = PDHG(f=f, g=g, operator=operator, max_iteration=10, gamma_fconj=0.5)
+        pdhg.run(verbose=0)
+        self.assertNotEqual(pdhg.sigma, sigma)
+        self.assertNotEqual(pdhg.tau, tau)   
+
```

Comment:
In this test you should also test that the values you input for the gammas are retained in the algorithm instance.